include/onnxruntime/core/common/common.h

```diff
@@ -253,13 +254,6 @@ void LogRuntimeError(uint32_t session_id, const common::Status& status, const ch
     }                                  \
   } while (0)
 
-// C++ Core Guideline check suppression.
-#if defined(_MSC_VER) && !defined(__NVCC__) && !defined(__clang__)
```

Comment:
Moved it to a new file because onnxruntime::Status also wants to use this macro. But this file includes "core/common/status.h"